docs/pages/components/date-picker/fragments/custom.md

```diff
@@ -5,29 +5,53 @@ import { DatePicker } from 'rsuite';
 import addDays from 'date-fns/addDays';
 import subDays from 'date-fns/subDays';
 
+const predefinedBottomRanges = [
+  {
+    label: 'yesterday',
+    value: addDays(new Date(), -1)
+  },
+  {
+    label: 'today',
+    value: new Date()
+  },
+  {
+    label: 'Prev Day',
+    closeOverlay: false,
+    value: date => {
+      return subDays(date, 1);
+    }
+  }
+];
+
+const predefinedRanges = [
+  {
+    label: 'yesterday',
+    value: addDays(new Date(), -1),
+    placement: 'left'
+  },
+  {
+    label: 'today',
+    value: new Date(),
+    placement: 'left'
+  },
+  {
+    label: 'Prev Day',
+    closeOverlay: false,
+    value: date => {
+      return subDays(date, 1);
+    }
+  }
+];
+
 const App = () => (
-  <div className="field">
+  <Stack direction="column" spacing={8} alignItems="flex-start">
```

Comment:
`Stack` should be imported, otherwise rendering will fail in stackblitz and codesandbox.

```diff
- import { DatePicker } from 'rsuite';
+ import { DatePicker, Stack } from 'rsuite';
```

Reply:
Nice spot. Added.